### PR TITLE
Wdired+wgrep

### PR DESCRIPTION
**Pull request by jojojames**
Would appreciate a quick look.

https://github.com/emacs-evil/evil-collection/issues/106

@Ambrevar @noctuid @terlar

### CODE REVIEW

**Comment by terlar** [comment]
Looks good! Thanks for the update.

One binding that I also have added to wgrep mode that might make sense is the `d` bound to `wgrep-mark-deletion`.

File: evil-collection-wdired.el [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+;; This program is distributed in the hope that it will be useful,
+;; but WITHOUT ANY WARRANTY; without even the implied warranty of
+;; MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+;; GNU General Public License for more details.
+
+;; You should have received a copy of the GNU General Public License
+;; along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+;;; Commentary:
+;;; Bindings for `wdired'
```
Comment (Ambrevar) [inline evil-collection-wdired.el]: Only two `;` here.

File: evil-collection-wgrep.el [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+;; This program is distributed in the hope that it will be useful,
+;; but WITHOUT ANY WARRANTY; without even the implied warranty of
+;; MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+;; GNU General Public License for more details.
+
+;; You should have received a copy of the GNU General Public License
+;; along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+;;; Commentary:
+;;; Bindings for `wgrep'
```
Comment (Ambrevar) [inline evil-collection-wgrep.el]: Same.

File: evil-collection-wdired.el (hunk 1 above)
Comment (Ambrevar) [reply evil-collection-wdired.el]: + missing period.

**Comment by Ambrevar** [comment]
Looks good to me.  I think something similar can be done for Helm buffers.  I'll look into it later.

**Comment by jojojames** [comment]
> One binding that I also have added to wgrep mode that might make sense is the d bound to wgrep-mark-deletion.

Hmnn, wouldn't you want to keep evil's normal d?

**Comment by jojojames** [comment]
I'll merge this now but we can continue talking about ^ if you are interested. @terlar

**Comment by terlar** [comment]
That is correct, probably not a good idea to change that behavior, I was thinking it didn't make sense to use `d`, but together with the text-objects it does. However the binding `dd` would make sense to run `wgrep-mark-deletion` as that is currently a noop in this mode (complains about buffer being read-only). I am not sure how to bind these motion keys though. What do you think? This would do what people expect.

**Pull request merged**